src/main/scala/nl.knaw.dans.easy.springfield/Smithers2.scala

```diff
@@ -71,11 +71,22 @@ trait Smithers2 {
     trace(avFile, requireTicket)
     val uri = path2Uri(avFile.resolve("properties").resolve("private"))
     debug(s"Smithers2 URI: $uri")
-    for {
-      response <- http("PUT", uri, requireTicket.toString)
-      if response.code == 200
-      _ <- checkResponseOk(response.body)
-    } yield ()
+    sendRequestAndCheckResponse(uri, "PUT")
+  }
+
+  /**
+   * Sets the playmode for playlist in a presentation. The path must point to the
+   * actual playlist resource.
+   *
+   * @param videoPlayListInPresentationPath path to the playlist in the presentation
+   * @param mode                            {menu|continuous} the to be played mode
+   * @return
+   */
+  def setPlayModeForVideoPlayListInPresentation(videoPlayListInPresentationPath: Path, mode: String): Try[Unit] = {
+    trace(videoPlayListInPresentationPath, mode)
+    val uri = path2Uri(videoPlayListInPresentationPath.resolve("properties").resolve("play-mode"))
+    println(s"Smithers2 URI: $uri")
```

Comment:
given the similar statements above and below, shouldn't this be

```suggestion
    debug(s"Smithers2 URI: $uri")
```